applications/nrf_desktop/configuration/nrf52840dk_nrf52840/prj_keyboard.conf

```diff
@@ -106,13 +106,6 @@ CONFIG_BT_CONN_TX_MAX=6
 CONFIG_BT_GATT_UUID16_POOL_SIZE=47
 CONFIG_BT_GATT_CHRC_POOL_SIZE=17
 
-CONFIG_BT_HIDS_INPUT_REP_MAX=3
-CONFIG_BT_HIDS_OUTPUT_REP_MAX=1
-CONFIG_BT_HIDS_FEATURE_REP_MAX=1
-CONFIG_BT_HIDS_ATTR_MAX=32
-CONFIG_BT_HIDS_DEFAULT_PERM_RW_ENCRYPT=y
-CONFIG_BT_CONN_CTX=y
-
```

Comment:
This configuration will get `DESKTOP_HIDS_FIRST_REPORT_DELAY` set to 500 even though it has it set to 0 now. Is it intentional?

Reply:
Yes, I think we could align behavior of all of our keyboards to prevent keypress drops on reconnections with dongle. :)